### PR TITLE
Remove "decimal version" warning emitted by babel-preset-env #265

### PR DESCRIPTION
**Pull request by yahyaKacem**
with the requested changes

### CODE REVIEW

File: src/config-parser.js [hunk 1]
```diff
@@ -360,7 +360,7 @@ function versionToFloat(ver) {
 
 function getElectronVersion(rootDir) {
   if (process.versions.electron) {
-    return versionToFloat(process.versions.electron);
+    return versionToFloat(process.versions.electron).toString();
```
Comment (paulcbetts) [inline src/config-parser.js]: Isn't this just `process.versions.electron`?

File: src/config-parser.js [hunk 2]
```diff
@@ -370,18 +370,18 @@ function getElectronVersion(rootDir) {
       // NB: lol this code
       let verRange = ourPkgJson.devDependencies[mod];
       let m = verRange.match(/(\d+\.\d+\.\d+)/);
-      if (m && m[1]) return m[1];
+      if (m && m[1]) return m[1].toString();
```
Comment (paulcbetts) [inline src/config-parser.js]: Isn't m[1] already a string?

File: src/config-parser.js [hunk 3]
```diff
@@ -370,18 +370,18 @@ function getElectronVersion(rootDir) {
...
     }
 
     try {
-      return process.mainModule.require(`${mod}/package.json`).version;
+      return process.mainModule.require(`${mod}/package.json`).version.toString();
```
Comment (paulcbetts) [inline src/config-parser.js]: Same deal

File: src/config-parser.js [hunk 4]
```diff
@@ -370,18 +370,18 @@ function getElectronVersion(rootDir) {
...
     } catch (e) {
       // NB: This usually doesn't work, but sometimes maybe?
     }
 
     try {
       let p = path.join(rootDir, mod, 'package.json');
-      return require(p).version;
+      return require(p).version.toString();
```
Comment (paulcbetts) [inline src/config-parser.js]: This too

File: src/config-parser.js [hunk 5]
```diff
@@ -391,7 +391,7 @@ function getElectronVersion(rootDir) {
     throw new Error("Can't automatically discover the version of Electron, you probably need a .compilerc file");
   }
 
-  return versionToFloat(version);
+  return versionToFloat(version).toString();
```
Comment (paulcbetts) [inline src/config-parser.js]: Same as above, we just want `version`

**Comment by malept** [comment]
I have a replacement PR in #268.

**Comment by malept** [comment]
Fixed in #268.

**Pull request closed without merging**